### ISSUE TITLE
window.onerror should be optionally null

### ISSUE THREAD

**Issue by Roaders** [microsoft/TypeScript#18198]
**TypeScript Version:**  2.4.0 / nightly (2.5.0-dev.201xxxxx)

**Code**

```ts

window.onerror(error); //should throw an error with strict null checking as might be null

window.onerror = null; //currently throws an error as cannot be null

```

**Expected behavior:**
I should be able to set `window.onerror` to null

**Actual behavior:**
I get a strict null check error

**Comment by mhegazy** [comment]
PRs welcomed. You can find more information about contributing lib.d.ts fixes at https://github.com/Microsoft/TypeScript/blob/master/CONTRIBUTING.md#contributing-libdts-fixes.

**Comment by Spongman** [comment]
not just `onerror`. pretty much every element property should be `|null`.

![image](https://user-images.githubusercontent.com/1088194/35901772-39344944-0b8d-11e8-889b-b1c6048a4767.png)

**Comment by saschanaz** [comment]
This has been fixed, right?

**Comment by jakebailey** [comment]
Just checking old issues; Yes, this one's definitely fixed; `onerror` can be null.
